src/main/java/io/kestra/plugin/gcp/runner/CloudRun.java

```diff
@@ -116,17 +116,17 @@
                   - id: shell
                     type: io.kestra.plugin.scripts.shell.Commands
                     inputFiles:
-                      data.txt: "{{inputs.file}}"
+                      data.txt: "{{ inputs.file }}"
                     outputFiles:
                       - out.txt
                     containerImage: centos
                     taskRunner:
-                      type: io.kestra.plugin.gcp.runner.GcpCloudRunTaskRunner
-                      projectId: "{{vars.projectId}}"
-                      region: "{{vars.region}}"
-                      bucket: "{{vars.bucker}}"
+                      type: io.kestra.plugin.gcp.runner.CloudRun
+                      projectId: "{{ vars.projectId }}"
+                      region: "{{ vars.region }}"
+                      bucket: "{{ vars.bucker }}"
                     commands:
-                    - cp {{workingDir}}/data.txt {{workingDir}}/out.txt""",
+                    - cp {{ workingDir }}/data.txt {{ workingDir }}/out.txt""",
```

Comment:
Adding whitespace like the azure PR

```suggestion
                      - cp {{ workingDir }}/data.txt {{ workingDir }}/out.txt""",
```